sims/benchmark-simuPOP.py

```diff
@@ -88,9 +88,9 @@ def fileopt(fname,opts):
 parser.add_argument("--popsize","-N", type=int, dest="popsize",
         help="size of each subpopulation",default=100)
 parser.add_argument("--theta","-ϴ", type=int, dest="theta",
-                    help="total mutation rate for whole chromosome: theta = 4 N u/site (sites)", default=100)
+                    help="total mutation rate for whole **diploid** chromosome: theta = 4 N u/site (sites)", default=100)
 parser.add_argument("--rho","-ρ", type=int, dest="rho",
```

Comment:
I made this thoughtless typo but also wanted to put down a comment as a reminder to check --- could there be a similar issue with specification of **mutation rates** in simuPOP?

Reply:
Results https://github.com/petrelharp/ftprime_ms/pull/11#issuecomment-343528484 suggest not